supervisor/docker/homeassistant.py

```diff
@@ -48,7 +48,7 @@ def ip_address(self) -> IPv4Address:
     @property
     def volumes(self) -> Dict[str, Dict[str, str]]:
         """Return Volumes for the mount."""
-        volumes = {}
+        volumes = {"/run/dbus": {"bind": "/run/dbus", "mode": "ro"}}
```

Comment:
This gives Home Assistant full dbus access right?

Reply:
Yes. however, they run already with full rights and on the host network. We should write a custom AppArmor profile for Core which can limit the access & scope also on dbus. But that is a challenge. I'm also fine comment core out of dbus and just forward with the other small code cleanup

Reply:
I think it's fine. Core can already talk to the supervisor anyway.

If we find integrations are messing it up, we will revisit.